frontend/src/components/Footer/StyledFooter.ts

```diff
@@ -32,63 +32,63 @@ const StyledFooter = styled.footer`
     }
   }
 
-  #logo {
+  #footer__logo {
```

Comment:
Here you have a id selector, but in BEM the focus is on class names selector(suggestion)